MainRepository/Program.cs

```diff
@@ -1,28 +1,34 @@
-﻿// Implementation of string is valid file path or not into tuple format.
+﻿// This program validates the file path
+// BNF diagram of finding valid file path
+// file://C:/WorkGIT/Assignments/MainRepository/Data/bnf-diagram.PNG

```

Comment:
This file link will work only for you as you will have the folder C:\WorkGIT\...